app/src/main/java/org/fossasia/susi/ai/activities/ForgotPasswordActivity.java

```diff
@@ -202,4 +161,45 @@ protected void onSaveInstanceState(Bundle outState) {
         outState.putCharSequenceArray("savedStates", values);
         outState.putBoolean("server",personalServer.isChecked());
     }
+
+    public void alertSuccess(Response<ForgotPasswordResponse> response) {
+        DialogInterface.OnClickListener dialogClickListener = new DialogInterface.OnClickListener() {
+            public void onClick(DialogInterface dialog, int id) {
+                Intent intent = new Intent(ForgotPasswordActivity.this, LoginActivity.class);
+                intent.setFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP | Intent.FLAG_ACTIVITY_NEW_TASK);
+                startActivity(intent);
+                finish();
+            }
+        };
+        alertTitle = null;
+        alertMessage = response.body().getMessage();
+        AlertboxHelper successAlertboxHelper = new AlertboxHelper(ForgotPasswordActivity.this,alertTitle,alertMessage,dialogClickListener,null,"Continue",null,Color.BLUE);
+        successAlertboxHelper.showAlertBox();
+    }
+
+    public void alertFailure() {
```

Comment:
First thing, alertFailure() , alertError() and alertException can be combined. Send title and message as parameters. Secondly, see these three functions are exactly same in login activity and forgot password activity. So, can't we just move that code in util and reuse it.